rskj-core/src/main/java/co/rsk/peg/FeePerKbElection.java

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
There's no need for this class. We can reuse `ABICallElection` for this. Can also get rid of `BridgeSerializationUtils::serializeFeePerKbElection` and `BridgeSerializationUtils::deserializeFeePerKbElection`.

Reply:
Please take a closer look. `ABICallElection` doesn't store the value being voted on. It also forces you to use an `ABICallSpec` that isn't really needed here when voting on a single value. It also doesn't have the functionality of resetting the election when there's a different vote.

I do think they share a common codebase that could be reused. My suggestion would be to create an issue to track this and perform a refactor in the future.

Reply:
Yes it does. It stores it on the `ABICallSpec`. An `ABICallSpec` represents a possible operation on the bridge. It stores a function name and any number of parameters encoded as byte arrays. That would allow encoding, for instance, setting a certain specific fee per kb. Then, the `ABICallElection` allows for voting on any number of possible `ABICallSpec`s. Specifically, you can also vote on a single value if you wanted.

Actually, a name change for the `ABICall` part on both the `ABICallElection` and `ABICallSpec` might also be benefitial. Maybe `Operation`? `OperationElection` and `OperationSpec`? This name is due to historical reasons.